M2/Macaulay2/m2/integers.m2

```diff
@@ -68,6 +69,7 @@ lcm(QQ,QQ) := (f,g) -> (
     d := gcd(f, g);
     if d == 0 then 0_QQ
     else abs f * (abs g / d))
+lcm ZZ := lcm QQ := identity
```

Comment:
Number, perhaps?

Reply:
I thought about that.  I stuck with `ZZ` and `QQ` since those are the ones we have binary methods for.  It seems like it might be a bit strange if, say, `gcd(3.0)` worked but `gcd(3.0, 6.0)` didn't.